### DISCUSSION TITLE
Does anybody else on Windows, Linux and Mac have issues with this specific Ubuntu torrent?

### DISCUSSION THREAD

**Discussion by Pentaphon** [transmission/transmission#4916] [Support]
I am hesitant to make an issue for this because it might just be me or the tracker or the torrent file so I am going to ask if anybody out there: does this torrent give you any issues?

Here's the steps I took. 

1) Download and clean install Transmission 4.0.0-dev (3ae8107ca9)
2) Start Transmission, don't change any settings at all and load this torrent:

https://releases.ubuntu.com/22.04/ubuntu-22.04.1-desktop-amd64.iso.torrent

from: https://ubuntu.com/download/alternative-downloads

3) It's been 10 minutes for me and I cannot connect to any peers at all. 

The only way I can get this torrent going after taking these steps is changing the port number up by 1 digit and restarting the client to let the torrent connect again. I very slowly start to finally get peers (1 peer at first, then 2, then 3, then 4, then 6 and so on) even though this torrent has 2,604 seeders and I finally start downloading at my normal torrent speeds after about a minute, which is still too long for a torrent of this health. 

Anybody else notice similar issues with this torrent? 

I'm using Windows 11 here, btw

**Comment by sweetppro** [comment]
it working fine for me. I get maximum d/l speed and many peers after a short time (macOS Ventura)

**Reply by Pentaphon** [reply]
OK thanks, that was expected since Mac users seem to have better performance on the reported issues than Windows users. Any Windows and Linux users out there?

**Reply by ckerr** [reply]
I just tested it on Linux and got lots of peers immediately.

**Reply by ckerr** [reply]
More broadly, maybe we need more tools on how to see what's going on with peers. https://github.com/transmission/transmission/pull/4900 is one step in that direction

**Comment by Pentaphon** [comment]
Still an issue with 4.0.3 on Windows and https://releases.ubuntu.com/23.04/ubuntu-23.04-desktop-amd64.iso.torrent

Might be an issue with their tracker but I've never seen a torrent be so slow to connect.

**Comment by ckerr** [comment]
This is being triaged in https://github.com/transmission/transmission/issues/6216
